crates/oxc_linter/src/snapshots/require_to_throw_message.snap

```diff
@@ -0,0 +1,37 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: require_to_throw_message
+---
+
+  ⚠ eslint-plugin-jest(require-to-throw-message): Require a message for `toThrow()`.
+   ╭─[require_to_throw_message.tsx:1:41]
+ 1 │ expect(() => { throw new Error('a'); }).toThrow();
+   ·                                         ───────
+   ╰────
+  help: Add an error message to "toThrow"()
+
+  ⚠ eslint-plugin-jest(require-to-throw-message): Require a message for `toThrow()`.
```

Comment:
Nitpick: looks like `toThrow` is not related to the current error, maybe `toThrowError`?

Reply:
Sorry, it was a mistake.
I have modified it to display the matcher related to errors.
c926297d16117121a3240f61a2b16edbf83cc51c